unison-src/transcripts/fuzzy-options.output.md

```diff
@@ -33,7 +53,7 @@ Definition args
 
 .> debug.fuzzy-options view _
 
-  Select a definition:
```

Comment:
Unfortunately generic articles are complicated in English (`a` vs `an`), but I think things read okay without it.

Reply:
I think it'll be good to improve this at some point to be consistent with the minimally-robotic UCM message style, but not going to delay the PR.